MessageApp/ConversationListViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ConversationListViewController.swift
+//  MessageApp
+//
+//  Created by Macbook on 25.09.2022.
+//
+
+import UIKit
+
+final class ConversationListViewController: UIViewController {
+    
+    private lazy var tableView = UITableView(frame: .zero)
+    
+    private let profileView = ProfileView()
+    
+    private let conversationCellModel = ConversationCellModel()
+    
+    private var onlineCells: [ConversationCell] = []
+    private var offlineCells: [ConversationCell] = []
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+            navigationItem.title = "Message App"
+        let profileButton = UIBarButtonItem(image: UIImage(systemName: "person.circle"), style: .plain, target: self, action: #selector(showProfile))
```

Comment:
Такие длинные строки лучше разносить в формате:
```suggestion
        let profileButton = UIBarButtonItem(
            image: UIImage(systemName: "person.circle"), 
            style: UIBarButtonItem.Style.plain,
            target: self, 
            action: #selector(showProfile)
        )
```

+ типы надо указывать польносью